interfaces/inc/c_pal/gballoc_hl.h

```diff
@@ -63,6 +63,8 @@ extern "C" {
 
     MOCKABLE_FUNCTION(, void, gballoc_hl_print_stats);
 
+    MOCKABLE_FUNCTION(, int, gballoc_hl_set_decay, int64_t, decay_milliseconds);

```

Comment:
>gballoc_hl_set_decay [](http://example.com/codeflow?start=29&length=20)

I think I would like more a set_option API. I know it would be an API over an API (or hiding API calls into an API call).
This is probably one case where that would help, like in the case of IOCTL.

If we have a set_option, the generic API of gballoc does not need to change every time we add such an option. Also all the implementations do not need to change, just the one that handles the option.

Reply:
Done